<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>kr_login</class>
 <widget class="QWidget" name="kr_login">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>420</width>
    <height>320</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>登录</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QPushButton[NotTitleBarBtn=true]
{	
	color:white;
	background-color:rgb(14 , 150 , 254);
	border: 1px solid rgb(11 , 137 , 234);
	border-radius:6px;
}

QPushButton:hover[NotTitleBarBtn=true]
{
	color:white;
	background-color:rgb(44 , 137 , 255);
	border: 1px solid rgb(11 , 137 , 234);
}

QPushButton:pressed[NotTitleBarBtn=true]
{
	color:white;
	background-color:rgb(14 , 135 , 228);
	border: 1px solid rgb(12 , 138 , 235);
	padding-left:3px;
	padding-top:3px;
}
QLineEdit[Account=true]
{
	font:12pt &quot;Microsoft YaHei&quot;;
    background:white;
    padding-left:5px ;
    padding-top:1px ;
    border-top-left-radius:5px;
    border-top-right-radius:5px;
    border: 1px solid rgb(209 , 209 , 209);
}
QLineEdit[Account=true]:hover
{
    padding-bottom:0px ;
    border: 1px solid rgb(21 , 131 , 221);
}

QLineEdit[Password=true]
{
	font:12pt &quot;Microsoft YaHei&quot;;
    background:white;
    padding-left:5px ;
    padding-top:1px ;
    border-bottom-left-radius:5px;
    border-bottom-right-radius:5px;
    border: 1px solid rgb(209 , 209 , 209);
    border-top:transparent;
}
QLineEdit[Password=true]:hover
{
    padding-top:0px ;
    border: 1px solid rgb(21 , 131 , 221);
}</string>
  </property>
  <widget class="QStackedWidget" name="stackedWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>7</y>
     <width>400</width>
     <height>301</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="LoginPage">
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>130</y>
       <width>401</width>
       <height>161</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="1,20,1">
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_2" stretch="6,2">
        <property name="spacing">
         <number>0</number>
        </property>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_6" stretch="1,1,1">
          <property name="spacing">
           <number>0</number>
          </property>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_8" stretch="3">
            <property name="spacing">
             <number>5</number>
            </property>
            <item>
             <widget class="QLineEdit" name="AccountLineEdit">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="echoMode">
               <enum>QLineEdit::Normal</enum>
              </property>
              <property name="placeholderText">
               <string>账号</string>
              </property>
              <property name="Account" stdset="0">
               <bool>true</bool>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_9" stretch="3">
            <property name="spacing">
             <number>2</number>
            </property>
            <item>
             <widget class="QLineEdit" name="PasswordLineEdit">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Expanding" vsizetype="Minimum">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="echoMode">
               <enum>QLineEdit::Password</enum>
              </property>
              <property name="placeholderText">
               <string>密码</string>
              </property>
              <property name="clearButtonEnabled">
               <bool>false</bool>
              </property>
              <property name="Password" stdset="0">
               <bool>true</bool>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_7" stretch="11,1">
            <item>
             <layout class="QVBoxLayout" name="verticalLayout" stretch="1">
              <property name="spacing">
               <number>0</number>
              </property>
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_5" stretch="1,1,1,1,1">
                <property name="spacing">
                 <number>7</number>
                </property>
                <item>
                 <widget class="QCheckBox" name="SaveCheckBox">
                  <property name="sizePolicy">
                   <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
                    <horstretch>0</horstretch>
                    <verstretch>0</verstretch>
                   </sizepolicy>
                  </property>
                  <property name="text">
                   <string>保存密码</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <spacer name="horizontalSpacer_9">
                  <property name="orientation">
                   <enum>Qt::Horizontal</enum>
                  </property>
                  <property name="sizeHint" stdset="0">
                   <size>
                    <width>40</width>
                    <height>20</height>
                   </size>
                  </property>
                 </spacer>
                </item>
                <item>
                 <widget class="QCheckBox" name="AutnLoginCheckBox">
                  <property name="sizePolicy">
                   <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
                    <horstretch>0</horstretch>
                    <verstretch>0</verstretch>
                   </sizepolicy>
                  </property>
                  <property name="text">
                   <string>自动登录</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <spacer name="horizontalSpacer_10">
                  <property name="orientation">
                   <enum>Qt::Horizontal</enum>
                  </property>
                  <property name="sizeHint" stdset="0">
                   <size>
                    <width>40</width>
                    <height>20</height>
                   </size>
                  </property>
                 </spacer>
                </item>
                <item>
                 <widget class="QCheckBox" name="AutonRunCheckBox">
                  <property name="sizePolicy">
                   <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
                    <horstretch>0</horstretch>
                    <verstretch>0</verstretch>
                   </sizepolicy>
                  </property>
                  <property name="text">
                   <string>开机自启</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
             </layout>
            </item>
            <item>
             <spacer name="verticalSpacer_2">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>20</width>
                <height>40</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,1">
          <property name="spacing">
           <number>9</number>
          </property>
          <item>
           <widget class="QPushButton" name="LoginBtn">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>登录</string>
            </property>
            <property name="shortcut">
             <string>Return</string>
            </property>
            <property name="NotTitleBarBtn" stdset="0">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="RegisterBtn">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>注册</string>
            </property>
            <property name="NotTitleBarBtn" stdset="0">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
    <widget class="QLabel" name="PitLabel">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>3</y>
       <width>401</width>
       <height>111</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="KTMlabel">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>40</y>
       <width>151</width>
       <height>61</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(255,255,255);
font: 11pt &quot;Forte&quot;;</string>
     </property>
     <property name="text">
      <string>KTM</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="LoginingPage">
    <widget class="QLabel" name="LoginingLabel">
     <property name="geometry">
      <rect>
       <x>107</x>
       <y>70</y>
       <width>186</width>
       <height>80</height>
      </rect>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>230</y>
       <width>401</width>
       <height>51</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_6" stretch="1,4,1">
      <item>
       <spacer name="horizontalSpacer_5">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="CancelBtn">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>取消</string>
        </property>
        <property name="shortcut">
         <string>Return</string>
        </property>
        <property name="NotTitleBarBtn" stdset="0">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_6">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QWidget" name="RegisterPage">
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>110</y>
       <width>401</width>
       <height>171</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="1,8,1">
      <item>
       <spacer name="horizontalSpacer_3">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_3" stretch="4,1">
        <property name="spacing">
         <number>6</number>
        </property>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_4" stretch="3,3,3,1,5">
          <property name="spacing">
           <number>0</number>
          </property>
          <item>
           <widget class="QLineEdit" name="R_AccountLineEdit">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="echoMode">
             <enum>QLineEdit::Normal</enum>
            </property>
            <property name="placeholderText">
             <string>账号</string>
            </property>
            <property name="Account" stdset="0">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="R_PasswordLineEdit">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Minimum">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="styleSheet">
             <string notr="true">QLineEdit
{
	font:12pt &quot;Microsoft YaHei&quot;;
    background:white;
    padding-left:5px ;
    padding-top:1px ;
    border: 1px solid rgb(209 , 209 , 209);
    border-top:transparent;
}
QLineEdit:hover
{
    padding-top:0px ;
    border: 1px solid rgb(21 , 131 , 221);
}</string>
            </property>
            <property name="echoMode">
             <enum>QLineEdit::Password</enum>
            </property>
            <property name="placeholderText">
             <string>密码</string>
            </property>
            <property name="clearButtonEnabled">
             <bool>false</bool>
            </property>
            <property name="Password" stdset="0">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="RA_PasswordLineEdit">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Minimum">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="echoMode">
             <enum>QLineEdit::Password</enum>
            </property>
            <property name="placeholderText">
             <string>再一次密码</string>
            </property>
            <property name="clearButtonEnabled">
             <bool>false</bool>
            </property>
            <property name="Password" stdset="0">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="verticalSpacer">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_10">
            <item>
             <widget class="QComboBox" name="FcomboBox">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="styleSheet">
               <string notr="true">QComboBox
{
color: rgb(0, 0, 0);
    background:rgba(250,250,250,0.7);
	border: 1px solid rgb(209 , 209 , 209);
	border-radius:3px;
}
 QComboBox::down-arrow {
	image: url(:/Resources/Login/combobox/downarrow.png);
 }
QComboBox::drop-down {
     subcontrol-origin:paddind;  
     subcontrol-position: top right;
     width: 29px;

     border-left-width: 1px;
     border-left-color: darkgray;
     border-left-style: solid; /* just a single line */
     border-top-right-radius: 3px; /* same radius as the QComboBox */
     border-bottom-right-radius: 3px;
 }
QComboBox:hover
{
    background:rgba(240,240,240,0.7);
	border: 1px solid rgb(209 , 209 , 209);
	border-radius:3px;
}
QComboBox:pressed
{
    background:rgba(230,230,230,0.7);
	border: 1px solid rgb(209 , 209 , 209);
	border-radius:3px;
    padding-left:3px;
    padding-top:3px;
}</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QComboBox" name="ScomboBox">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="styleSheet">
               <string notr="true">QComboBox
{
color: rgb(0, 0, 0);
    background:rgba(250,250,250,0.7);
	border: 1px solid rgb(209 , 209 , 209);
	border-radius:3px;
}
 QComboBox::down-arrow {
	image: url(:/Resources/Login/combobox/downarrow.png);
 }
QComboBox::drop-down {
     subcontrol-origin:paddind;  
     subcontrol-position: top right;
     width: 29px;

     border-left-width: 1px;
     border-left-color: darkgray;
     border-left-style: solid; /* just a single line */
     border-top-right-radius: 3px; /* same radius as the QComboBox */
     border-bottom-right-radius: 3px;
 }
QComboBox:hover
{
    background:rgba(240,240,240,0.7);
	border: 1px solid rgb(209 , 209 , 209);
	border-radius:3px;
}
QComboBox:pressed
{
    background:rgba(230,230,230,0.7);
	border: 1px solid rgb(209 , 209 , 209);
	border-radius:3px;
    padding-left:3px;
    padding-top:3px;
}</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <item>
           <widget class="QPushButton" name="R_RegisterBtn">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>注册</string>
            </property>
            <property name="shortcut">
             <string>Return</string>
            </property>
            <property name="NotTitleBarBtn" stdset="0">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="R_RegisterGVBtn">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>返回</string>
            </property>
            <property name="NotTitleBarBtn" stdset="0">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <spacer name="horizontalSpacer_4">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>50</y>
       <width>401</width>
       <height>51</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_7">
      <item>
       <spacer name="horizontalSpacer_7">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_5">
        <item>
         <widget class="QLabel" name="InfoLabel">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>账号注册长度20字符以内，支持英文数字_,.-</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="InfoLabel2">
          <property name="text">
           <string>密码注册长度6到20字符，支持英文数字_,.-</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <spacer name="horizontalSpacer_8">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </widget>
  </widget>
  <widget class="QPushButton" name="CloseBtn">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>10</y>
     <width>30</width>
     <height>30</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton
{
	background-color: transparent;
	border-image: url(:/Resources/Login/label/close.png);
}
QPushButton：hover
{
	background-color: transparent;
	border-image: url(:/Resources/Login/label/closeHover.png);
}
QPushButton：hover
{
	background-color: rgba(116, 116, 116,0.4);
	padding-left:3px;
	padding-top:3px;
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
